shared/ui-components/src/forms/CheckboxGroup.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+
+export const CheckboxGroup = ({ groupLabel, fields }: Props) => {
+  return (
+    <div>
+      <p className="field-label--caps">{groupLabel}</p>
+
+      <div className="mt-3">
+        {fields &&
+          fields.map((item) => (
+            <div className="field" key={item.name}>
+              <input
+                type="checkbox"
+                id={item.name}
+                name={item.name}
+                defaultChecked={item.defaultChecked || false}
+                ref={item.register}
+              />
+              <label htmlFor={item.name} className="font-semibold">
+                {item.label}
+              </label>
+            </div>
+          ))}
+      </div>
+    </div>
+  )
+}
+
+interface CheckboxSingle {
```

Comment:
Just an FYI: our convention typically has been to put interface declarations above components in a file